virtcontainers/hypervisor.go

```diff
@@ -77,7 +77,7 @@ const (
 	vSockLogsPort = 1025
 
 	// MinHypervisorMemory is the minimum memory required for a VM.
-	MinHypervisorMemory = 256
+	MinHypervisorMemory = 8
```

Comment:
Maybe we should make it per-hypervisor default?

Reply:
I’m not sure if we should even have a minimum requirement, to be honest. 8 seemed reasonably small for each of them.

Reply:
I think you could remove the min.  Finding min values are hard and can always be tricky. You can just let the underlying system fail and propagate the error up to the user and not be in the game of trying to find a min value that works for everyone.

Reply:
Let me checkout the error that'll end up propagating. Agreed that a minimum doesn't buy us much;  the admin should be able to 'do the right thing'

Reply:
@egernst I am in agreement with @crosbymichael here. Lets remove the min memory here in light of Pod overhead feature now.